### PR TITLE
Update tikzit to 1.1

### PR DESCRIPTION
**Pull request by gtmgianni**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/tikzit.rb [hunk 1]
```diff
@@ -2,11 +2,12 @@
   version '1.1'
   sha256 '7e00a64fe5c96963ae5b09f1f59698b8b487239c53eed01a3bf012cf024f0686'
 
+  # downloads.sourceforge.net/tikzit was verified as official when first introduced to the cask
   url "https://downloads.sourceforge.net/tikzit/TikZiT-osx-#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/tikzit.rb]: The current download URL on the homepage is from `bintray`.

Comment (gtmgianni) [reply Casks/tikzit.rb]: The `sha256` for the file from `bintray` is different though...

Comment (gtmgianni) [reply Casks/tikzit.rb]: Not sure why, I thought we could stick with this URL to save the trouble of verifying the `sha256`.

Comment (vitorgalvao) [reply Casks/tikzit.rb]: But as it is, none of the official sources link to this link on sourceforge, so we cannot claim it as official anymore.

Change it to the bintray one. We‘re changing most of the important things about this cask, like `home` and `appcast`, we can consider it makes sense for them to have altered the package as well and not require verification.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**